jsonld.js/src/test/scala/org/w3/banana/jsonldjs/io/JsonLdJsParserTest.scala

```diff
@@ -0,0 +1,39 @@
+package org.w3.banana
+package jsonldjs
+package io
+
+import zcheck.SpecLite
+import scalajs.concurrent.JSExecutionContext.Implicits.runNow
+import org.w3.banana.plantain._
+
+object JsonLdJsParserTest extends SpecLite {
```

Comment:
Why not just port over the JsonLd test (plus its dependencies)? At least then we are not adding an issue that this test will have to be deprecated/moved